ios/FluentUI/Vnext/Avatar/AvatarTokens.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 @objc public enum MSFAvatarStyle: Int, CaseIterable {
     case `default`
     case accent
-    case group
+    case square
```

Comment:
I see. I renamed it because the tokens I created would be named MSFAvatarGroupTokens which I felt were too similar to MSFGroupAvatarTokens? It's definitely not a permanent change and I'd love to discuss other names to use

Reply:
I think we should keep the style name as "group" since it's what matches the figma definition by the designers. In case we redefine names later, we can have a PR just for the renaming of types.